jenkins-jobs/job-dsl/ocp_downstream_apicurio_prepare.groovy

```diff
@@ -0,0 +1,30 @@
+pipelineJob('ocp-downstream-apicurio-prepare-job') {
+    displayName('Apicurio deployment preparation')
+    description('Republishes apicurio registry images to quay.io and replaces them in deployment descriptor')
+
+    properties {
+        githubProjectUrl('https://github.com/debezium/debezium')
+    }
+
+    parameters {
+//        QUAY CONFIG
+        stringParam('QUAY_CREDENTIALS', 'rh-integration-quay-creds', "Quay.io credentials id")
+        stringParam('QUAY_ORGANISATION', '', "Organisation where images are copied")
```

Comment:
Please make quotes consistent and use ``'`` where possible. You can also do so in the strimzi counterpart.